webapp/migrations/Version20240629154640.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240629154640 extends AbstractMigration
+{
+    private const NEW_ROLES = ['api_problem_change' => 'API Problem Changer',
+                               'api_contest_change' => 'API Contest Changer'];
+
+    public function getDescription(): string
+    {
+        return 'Add new roles to the database.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        foreach(self::NEW_ROLES as $role => $description) {
+            $existingRole = $this->connection->fetchAssociative('SELECT * FROM role WHERE role = :role', ['role' => $role]);
```

Comment:
When would the role already exist? You can't create it in the UI so I'm not sure if we should handle this case?

Reply:
I suspected we load the default fixture before the migrations.

Reply:
I don't think so. Since then some tables might still be wrong

Reply:
I checked and you're right. Added a commit for it.